mail/permissions.py

```diff
@@ -101,3 +103,30 @@ def has_object_permission(self, request, view, obj):
         )
         course_run_keys = obj.courserun_set.values_list('edx_course_key', flat=True)
         return any([mmtrack.has_paid(course_run_key) for course_run_key in course_run_keys])
+
+
+class MailGunWebHookPermission(BasePermission):
```

Comment:
Please make sure to have a test for this

Reply:
Added